modules/nw-multinetwork-sriov.adoc

```diff
@@ -12,22 +12,44 @@ endif::openshift-origin[]
 [id="nw-multinetwork-sriov_{context}"]
 = Configuring SR-IOV
 
-{product-title} includes the capability to use SR-IOV hardware on
-{product-title} nodes, which enables you to attach SR-IOV virtual function (VF)
-interfaces to Pods in addition to other network interfaces.
+{product-title} includes the capability to use SR-IOV hardware on your nodes.
+You can attach SR-IOV virtual function (VF) interfaces to Pods in addition to
+other network interfaces.
 
-Two components are required to provide this capability: the SR-IOV network
-device plug-in and the SR-IOV CNI plug-in.
+You can use the {product-title} console to install SR-IOV, by deploying, the
+SR-IOV Network Operator.  The SR-IOV Network Operator creates and manages the
+components of the SR-IOV stack. The Operator provides following features:
+
+* Discover the SR-IOV network device in cluster.
+* Initialize the supported SR-IOV NIC models on nodes.
+* Provision the SR-IOV network device plug-in on nodes.
+* Provision the SR-IOV CNI plug-in executable on nodes.
+* Provision the Network Resources Injector in cluster.
+* Manage configuration of SR-IOV network device plug-in.
+* Generate NetworkAttachmentDefinition custom resources for the SR-IOV CNI
+plug-in.
+
+Here's the function of each abovementioned SR-IOV components .
 
 * The SR-IOV network device plug-in is a Kubernetes device plug-in for
 discovering, advertising, and allocating SR-IOV network virtual function (VF)
 resources. Device plug-ins are used in Kubernetes to enable the use of limited
 resources, typically in physical devices. Device plug-ins give the Kubernetes
-scheduler awareness of which resources are exhausted, allowing Pods to be
+scheduler awareness of which resources are exhausted, allowing pods to be
 scheduled to worker nodes that have sufficient resources available.
 
 * The SR-IOV CNI plug-in plumbs VF interfaces allocated from the SR-IOV device
-plug-in directly into a Pod.
+plug-in directly into a pod.
+
+* The Network Resources Injector is a Kubernetes Dynamic Admission Controller
+Webhook, which provides functionality of patching Kubernetes pod specifications
+with requests and limits of custom network resources such as SR-IOV VFs.
+
+[NOTE]
+====
+The SR-IOV admission controller is enabled by default, and cannot be disabled by
```

Comment:
s/SR-IOV admission controller/Network Resources Injector/g

I forgot to update the name in NOTE.

Reply:
Updated.